packages/analytics-js-integrations/src/integrations/MicrosoftClarity/browser.js

```
@@ -55,7 +56,11 @@ class MicrosoftClarity {
     if (context?.traits?.customPageId) {
       customPageId = context.traits.customPageId;
     }
-    window.clarity('identify', userId, sessionId, customPageId);
+
+    const identifyPromise = window.clarity('identify', userId, sessionId, customPageId);
+    if (!!identifyPromise) { // Clarity SDK is ready
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)